spec/onc_certification_g10_test_kit/bulk_data_group_export_spec.rb

```diff
@@ -256,6 +258,16 @@ def run(runnable, inputs = {})
 
       expect(result.result).to eq('pass')
     end
+
+    it 'sends content type headers' do
+      get_stub = stub_request(:get, polling_url.to_s)
+        .with(headers: { 'Authorization' => "Bearer #{bearer_token}" })
+        .with(headers: { 'Accept' => 'application/json' })
```

Comment:
Why not just put both in one hash?  I'm a bit concerned the second will clobber the first...  i did a quick scan of the documentation and didnt see anything on this.

Reply:
Yeah, probably safer all in one, pushed.